libc/newhdrgen/yaml_to_classes.py

```diff
@@ -0,0 +1,144 @@
+#!/usr/bin/env python
+#
+# ===- Generate headers for libc functions  -------------------*- python -*--==#
+#
+# Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+#
+# ==-------------------------------------------------------------------------==#
+
+
+import yaml
+import re
+import argparse
+
+from pathlib import Path
+from header import HeaderFile
+from class_implementation.classes.macro import Macro
```

Comment:
you should make sure to land a patch adding these classes before landing this patch. Ideally the code in the repository should always work, even if it's not complete yet.

Reply:
That patch should be ready to land today.